packages/core/utils/src/convert-query-params.ts

```diff
@@ -386,11 +390,33 @@ const createTransformer = ({ getModel }: TransformerOptions) => {
           );
         }
 
+        /**
+         * Validate nested population queries in the context of a polymorphic attribute (dynamic zone, morph relation).
+         *
+         * If 'populate' exists in subPopulate, its value should be constrained to a wildcard ('*').
+         *
+         * TODO: This approach should be marked as deprecated and replaced with a more robust solution in future versions.
```

Comment:
why a todo ? can't you add a log now ?

Reply:
By log you mean adding a server log every time a `populate='*' `is used for a DZ? I fear it could spam the logs too much :thinking:

Reply:
True, then should you just remove the TODO ? you already set it as deprecated in the typings

Reply:
The todo was more about the replacing part, but I can remove it altogether